main.py

```diff
@@ -0,0 +1,52 @@
+import json
+from dataclasses import dataclass, asdict, field
+from datetime import datetime
+from pathlib import Path
+
+# 📁 File where journal entries are stored
+DB_FILE = Path("journal.json")
+DB_FILE.touch(exist_ok=True)
+
+# 📝 Dataclass representing a journal entry
+@dataclass
+class JournalEntry:
+    title: str
+    content: str
+    date: str = field(default_factory=lambda: datetime.now().isoformat())
+
+# 📥 Load entries from the file
+def load_entries():
+    if DB_FILE.stat().st_size == 0:
```

Comment:
nice early return, could also rely on below not incurring the extra list comprehension if data is empty (I think)